src/app/modules/customer-management/store/customer-management.selectors.ts

```diff
@@ -12,3 +12,13 @@ export const getStatusMessage = createSelector(getCustomerState, (messageState)
 export const allCustomers = createSelector(getCustomerState, (state: CustomerState) => {
   return state.data;
 });
+
+export const customerIdtoEdit = createSelector(getCustomerState, (state: CustomerState) => {
+  return state.customerIdToEdit;
+});
+
+export const getCustomerById = createSelector(allCustomers, customerIdtoEdit, (customers, customerId) => {
+  return customers.find((customer) => {
```

Comment:
could add the validation for (customers, customerId)

Reply:
done.

Reply:
All changes were applied. @enriquezrene when you have a free time, please, check it.